lib/commands/createproduct.js

```diff
@@ -70,7 +70,7 @@ function createProduct(opts,request,done){
 	}
 
 	product.name = opts.productName
-	product.displayName = opts.productName
+	product.displayName = opts.displayName
```

Comment:
It looks like this value will always be null as the [descriptor](https://github.com/apigee/apigeetool-node/blob/d165bbc6d66194a0ad7efba85d718ecd7f6ebc2a/lib/commands/createproduct.js#L10) that declares the CLI options doesn't include `displayName` as one of the options. Has this been tested?

Reply:
added `displayName` options in CLI options.